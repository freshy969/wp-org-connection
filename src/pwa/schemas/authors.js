import { schema } from 'normalizr';

export const author = new schema.Entity(
  'author',
  {},
  {
    processStrategy(entity) {
      entity.mst = 'author';
      entity.type = 'author';
      return entity;
    },
  },
);
export const authors = new schema.Array(author);
