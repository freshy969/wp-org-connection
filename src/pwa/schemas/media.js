import { schema } from 'normalizr';

export const media = new schema.Entity('media');
export const mediaArray = new schema.Array(media);
