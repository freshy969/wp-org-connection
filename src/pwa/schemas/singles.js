import { schema } from 'normalizr';

const author = new schema.Entity('author');
const media = new schema.Entity('media');
const taxonomy = new schema.Entity('taxonomy');
const taxonomies = new schema.Array(new schema.Array(taxonomy));
export const single = new schema.Entity('single')
single.define({
  _embedded: {
    author: [author],
    'wp:featuredmedia': [media],
    'wp:term': taxonomies,
    up: single,
  },
})
export const singles = new schema.Array(single);
