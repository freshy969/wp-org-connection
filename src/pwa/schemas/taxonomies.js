import { schema } from 'normalizr';

export const taxonomy = new schema.Entity('taxonomy');
export const taxonomies = new schema.Array(taxonomy);
