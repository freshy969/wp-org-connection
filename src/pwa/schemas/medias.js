import { schema } from 'normalizr';
import { author } from './authors';

export const media = new schema.Entity(
  'media',
  {
    _embedded: {
      author: [author],
    },
  },
  {
    processStrategy(entity) {
      entity.mst = 'media';
      return entity;
    },
  },
);

export const medias = new schema.Array(media);
