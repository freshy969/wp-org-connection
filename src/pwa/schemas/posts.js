/* eslint-disable no-underscore-dangle */
import { schema } from 'normalizr';
import { taxonomy } from './taxonomies';
import { author } from './authors';

const media = new schema.Entity('media');
const taxonomies = new schema.Array(new schema.Array(taxonomy));
export const post = new schema.Entity(
  'post',
  {},
  {
    processStrategy(entity) {
      // Get all taxonomies and generate a map so we can now later on which props are actually
      // taxonomies.
      if (entity._embedded && entity._embedded['wp:term']) {
        const taxonomiesMap = {};
        entity._embedded['wp:term'].forEach(term =>
          term.forEach(item => {
            const type = item.taxonomy === 'post_tag' ? 'tag' : item.taxonomy;
            taxonomiesMap[type] = taxonomiesMap[type] || [];
            taxonomiesMap[type].push(item.id);
          }),
        );
        return { ...entity, taxonomiesMap };
      }
      return entity;
    },
  },
);
post.define({
  _embedded: {
    author: [author],
    'wp:featuredmedia': [media],
    'wp:term': taxonomies,
    up: post,
  },
});
export const posts = new schema.Array(post);
