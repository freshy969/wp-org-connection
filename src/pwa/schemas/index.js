import { post, posts } from './posts';
import { taxonomy, taxonomies } from './taxonomies';
import { author, authors } from './authors';

export {
  post,
  posts,
  taxonomy,
  taxonomies,
  author,
  authors,
};
