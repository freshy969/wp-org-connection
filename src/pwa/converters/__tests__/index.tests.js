import { normalize } from 'normalizr';
import convert from '../';
import post60 from '../../__tests__/post-60.json';
import media212 from '../../__tests__/media-212.json';
import page184 from '../../__tests__/page-with-subpage.json';
import { entity } from '../../schemas';

const { entities: entitiesFromPost60 } = normalize(post60, entity);

test('Convert post', () => {
  expect(convert(entitiesFromPost60.single[60])).toMatchSnapshot();
});

test('Convert category', () => {
  expect(convert(entitiesFromPost60.taxonomy[3])).toMatchSnapshot();
});

test('Convert tag', () => {
  expect(convert(entitiesFromPost60.taxonomy[9])).toMatchSnapshot();
});

test('Convert author', () => {
  expect(convert(entitiesFromPost60.author[4])).toMatchSnapshot();
});

test('Convert media from post', () => {
  expect(convert(entitiesFromPost60.media[62])).toMatchSnapshot();
});

const { entities: entitiesFromMedia212 } = normalize(media212, entity);
test('Convert media', () => {
  expect(convert(entitiesFromMedia212.media[212])).toMatchSnapshot();
});

const { entities: entitiesFromPage184 } = normalize(page184, entity);
test('Convert page', () => {
  expect(convert(entitiesFromPage184.single[184])).toMatchSnapshot();
});
