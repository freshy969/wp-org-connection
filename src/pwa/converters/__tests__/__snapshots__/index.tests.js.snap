// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Convert author 1`] = `
Object {
  "avatar": "https://secure.gravatar.com/avatar/eebc7d67bdc645559fd6634e0335a42b",
  "description": "Alan is a WordPress enthusiast who enjoys travelling, music and pizza.",
  "id": 4,
  "link": "https://demo.worona.org/author/alan/",
  "mst": "author",
  "name": "Alan Martin",
  "slug": "alan",
  "type": "author",
}
`;

exports[`Convert category 1`] = `
Object {
  "headMeta": Object {
    "title": "Photography Archives - Demo Worona",
  },
  "id": 3,
  "link": "https://demo.worona.org/wp-cat/photography/",
  "mst": "taxonomy",
  "name": "Photography",
  "slug": "photography",
  "target": undefined,
  "type": "category",
}
`;

exports[`Convert media 1`] = `
Object {
  "alt": "",
  "author": 6,
  "caption": "",
  "creationDate": 1498558207000,
  "description": undefined,
  "id": 212,
  "link": "https://demo.worona.org/audio-test/peter-griffin-tada/",
  "mediaType": "file",
  "meta": Object {
    "title": "peter-griffin-tada - Demo Worona",
  },
  "mimeType": "audio/mpeg",
  "mst": "media",
  "original": Object {
    "filename": undefined,
    "height": undefined,
    "url": "https://demo.worona.org/wp-content/uploads/2017/06/Peter-Griffin-TADA.mp3",
    "width": undefined,
  },
  "sizes": Array [],
  "slug": "peter-griffin-tada",
  "title": "peter-griffin-tada",
  "type": "media",
}
`;

exports[`Convert media from post 1`] = `
Object {
  "alt": "iceland",
  "author": 2,
  "caption": undefined,
  "creationDate": 946681200000,
  "description": undefined,
  "id": 62,
  "link": "https://demo.worona.org/the-beauties-of-gullfoss/iceland-test/",
  "mediaType": "image",
  "meta": Object {
    "title": "iceland - Demo Worona",
  },
  "mimeType": "image/jpeg",
  "mst": "media",
  "original": Object {
    "filename": "2016/11/Iceland-test.jpg",
    "height": 3025,
    "url": "https://demo.worona.org/wp-content/uploads/2016/11/Iceland-test.jpg",
    "width": 5000,
  },
  "sizes": Array [
    Object {
      "filename": "Iceland-test-150x150.jpg",
      "height": 150,
      "url": "https://demo.worona.org/wp-content/uploads/2016/11/Iceland-test-150x150.jpg",
      "width": 150,
    },
    Object {
      "filename": "Iceland-test-300x182.jpg",
      "height": 182,
      "url": "https://demo.worona.org/wp-content/uploads/2016/11/Iceland-test-300x182.jpg",
      "width": 300,
    },
    Object {
      "filename": "Iceland-test-768x465.jpg",
      "height": 465,
      "url": "https://demo.worona.org/wp-content/uploads/2016/11/Iceland-test-768x465.jpg",
      "width": 768,
    },
    Object {
      "filename": "Iceland-test-1024x620.jpg",
      "height": 620,
      "url": "https://demo.worona.org/wp-content/uploads/2016/11/Iceland-test-1024x620.jpg",
      "width": 1024,
    },
    Object {
      "filename": "Iceland-test-1200x726.jpg",
      "height": 726,
      "url": "https://demo.worona.org/wp-content/uploads/2016/11/Iceland-test-1200x726.jpg",
      "width": 1200,
    },
    Object {
      "filename": "Iceland-test.jpg",
      "height": 3025,
      "url": "https://demo.worona.org/wp-content/uploads/2016/11/Iceland-test.jpg",
      "width": 5000,
    },
  ],
  "slug": "iceland-test",
  "title": "iceland",
  "type": "media",
}
`;

exports[`Convert page 1`] = `
Object {
  "author": 1,
  "content": "<h5>If you are just curious about our story you&#8217;re in the right spot üôÇ</h5>
<p>This site features articles on cities in Europe and around the world. We‚Äôre passionate about travelling‚Äî doing something fun, seeing cool things, and meeting different people.</p>
<blockquote><p>‚ÄúTwenty years from now you will be more disappointed by the things you didn‚Äôt do than by the ones you did do.‚Äù ‚Äì Mark Twain</p></blockquote>
<p><img class=\\"alignnone size-full wp-image-185\\" src=\\"https://demo.worona.org/wp-content/uploads/2017/03/pexels-photo-297755.png\\" alt=\\"about-us-team\\" width=\\"700\\" height=\\"458\\" srcset=\\"https://demo.worona.org/wp-content/uploads/2017/03/pexels-photo-297755.png 700w, https://demo.worona.org/wp-content/uploads/2017/03/pexels-photo-297755-300x196.png 300w\\" sizes=\\"(max-width: 700px) 85vw, 700px\\" /></p>
<h6><em>Alan Martin</em></h6>
<p>Alan is a blogger and WordPress enthusiast. He loves¬†travelling, music and pizza.¬†Check out his latest post about¬†<a href=\\"https://demo.worona.org/sunrise-at-phu-quoc-island/\\">Ph√∫ Qu·ªëc Island (Vietnam)</a>.</p>
<h6><em>Reyes Mart√≠nez</em></h6>
<p>Reyes loves travelling, photography and¬†writing. Don&#8217;t miss her latest post about the amazing <a href=\\"https://demo.worona.org/small-and-picturesque-french-villages/\\">Picturesque French Villages</a>.</p>
<h6><em>Luis Herranz</em></h6>
<p>Luis is an entrepreneur that enjoys travelling and reading. In his latest post, he writes about India and The <a href=\\"https://demo.worona.org/latest-trip-to-india/\\">Taj Mahal</a>,¬†one of the universally admired masterpieces of the world‚Äôs heritage.</p>
<p><img class=\\"alignnone size-full wp-image-193\\" src=\\"https://demo.worona.org/wp-content/uploads/2017/03/road-trip.png\\" alt=\\"road-trip\\" width=\\"700\\" height=\\"467\\" srcset=\\"https://demo.worona.org/wp-content/uploads/2017/03/road-trip.png 700w, https://demo.worona.org/wp-content/uploads/2017/03/road-trip-300x200.png 300w\\" sizes=\\"(max-width: 700px) 85vw, 700px\\" /></p>
<p>This is just a demo blog, but you might want to check out some of the really good travel bloggers out there:</p>
<ul>
<li><strong>Nomadic Matt</strong>. Each month over 1.000.000 people use his advice to travel better, cheaper, and longer.</li>
<li><strong>Expert Vagabond</strong> by Matt &#8211; adventure travel photographer and professional blogger.</li>
<li><strong>The Aussie Nomad</strong>¬†by Chris. He has been living abroad as an expat and exploring the world since the beginning of 2010.</li>
</ul>
<p>You can see the¬†full list of top travel blogs¬†<a href=\\"http://www.theexpeditioner.com/the-top-50-travel-blogs/\\">here</a>.</p>
<h4>See you down the road!</h4>
<p><iframe style=\\"border: 0;\\" src=\\"https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d31758.896101501236!2d13.36989231715827!3d52.510655459236325!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x47a84e373f035901%3A0x42120465b5e3b70!2sBerl%C3%ADn%2C+Alemania!5e0!3m2!1ses!2ses!4v1490629160374\\" width=\\"600\\" height=\\"450\\" frameborder=\\"0\\" allowfullscreen=\\"allowfullscreen\\"></iframe></p>
",
  "creationDate": 1490627362000,
  "excerpt": "<p>If you are just curious about our story you&#8217;re in the right spot üôÇ This site features articles on cities in Europe and around the world. We‚Äôre passionate about travelling‚Äî doing something fun, seeing cool things, and meeting different people. ‚ÄúTwenty years from now you will be more disappointed by the things you didn‚Äôt do &hellip; <a href=\\"https://demo.worona.org/audio-test/about-us/\\" class=\\"more-link\\">Continue reading<span class=\\"screen-reader-text\\"> &#8220;About us&#8221;</span></a></p>
",
  "featured": null,
  "headMeta": Object {
    "title": "About us - Demo Worona",
  },
  "id": 184,
  "link": "https://demo.worona.org/audio-test/about-us/",
  "modificationDate": 1521819351000,
  "mst": "single",
  "parent": 211,
  "slug": "about-us",
  "target": undefined,
  "taxonomies": Object {
    "latest": Array [
      "page",
    ],
  },
  "title": "About us",
  "type": "page",
}
`;

exports[`Convert post 1`] = `
Object {
  "author": 4,
  "content": "<p>Gullfoss is a waterfall located in the canyon of the Hv√≠t√° river in southwest Iceland.</p>
<p><a href=\\"https://en.wikipedia.org/wiki/Gullfoss\\">Gullfoss</a> is one of the most popular <a href=\\"https://demo.worona.org/iceland/\\">tourist attractions in Iceland</a>. The wide Hv√≠t√° rushes southward, and about a kilometre above the falls it turns sharply to the right and flows down into a wide curved three-step ‚Äústaircase‚Äù and then abruptly plunges in two stages (11 metres or 36 feet, and 21 metres or 69 feet) into a crevice 32 metres (105 ft) deep. The crevice, about 20 metres (66 ft) wide and 2.5 kilometres (1.6 mi) in length, extends perpendicular to the flow of the river. The average amount of water running down the waterfall is 140 cubic metres (4,900 cu ft) per second in the summer and 80 cubic metres (2,800 cu ft) per second in the winter. The <a href=\\"https://demo.worona.org/sample-page/\\">highest flood</a> measured was 2,000 cubic metres (71,000 cu ft) per second.</p>
<h3>An iconic waterfall of Iceland</h3>
<figure id=\\"attachment_70\\" style=\\"width:840px;\\" class=\\"wp-caption alignnone\\"><img class=\\"wp-image-70 size-large\\" src=\\"https://demo.worona.org/wp-content/uploads/2016/11/Gullfoss-1024x577.jpg\\" alt=\\"gullfoss\\" width=\\"840\\" height=\\"473\\"><figcaption class=\\"wp-caption-text\\">Gullfoss, an iconic waterfall of Iceland.</figcaption></figure>

<div class=\\"4\\">During the first half of the 20th century and some years into the late 20th century, there was much speculation about using Gullfoss to generate electricity. During this period, the waterfall was rented indirectly by its owners, T√≥mas T√≥masson and Halld√≥r Halld√≥rsson, to foreign investors. However, the investors‚Äô attempts were unsuccessful, partly due to lack of money. The waterfall was later sold to the state of Iceland, and is now protected.</div>

<blockquote>
<p style=\\"text-align:left;\\">Iceland is not a destination. It‚Äôs an adventure set in stunning scenery.</p>
</blockquote>
<p>Sigr√≠√∞ur T√≥masd√≥ttir, the daughter of T√≥mas T√≥masson, was determined to preserve the waterfall‚Äôs condition and even threatened to throw herself down. Although it is widely believed, the very popular story that Sigr√≠√∞ur saved the waterfall from exploitation is untrue. <a href=\\"http://www.nat.is/travelguideeng/gullfoss_sigridur_tomasdottir.htm\\">A stone memorial to Sigri√∞ur</a>, located above the falls, depicts her profile.</p>
<p><iframe width=\\"840\\" height=\\"473\\" src=\\"https://www.youtube.com/embed/3bUtvUAeX8s?feature=oembed\\" frameborder=\\"0\\" allowfullscreen></iframe></p>

<ul><li><a title=\\"Geography of Iceland\\" href=\\"https://en.wikipedia.org/wiki/Geography_of_Iceland\\">Geography of Iceland</a></li>
<li><a title=\\"Iceland plume\\" href=\\"https://en.wikipedia.org/wiki/Iceland_plume\\">Iceland plume</a></li>
<li><a title=\\"List of lakes of Iceland\\" href=\\"https://en.wikipedia.org/wiki/List_of_lakes_of_Iceland\\">List of lakes of Iceland</a></li>
<li><a class=\\"mw-redirect\\" title=\\"List of islands off Iceland\\" href=\\"https://en.wikipedia.org/wiki/List_of_islands_off_Iceland\\">List of islands off Iceland</a></li>
<li><a title=\\"List of volcanoes in Iceland\\" href=\\"https://en.wikipedia.org/wiki/List_of_volcanoes_in_Iceland\\">List of volcanoes in Iceland</a></li>
<li><a title=\\"List of waterfalls\\" href=\\"https://en.wikipedia.org/wiki/List_of_waterfalls\\">List of waterfalls</a></li>
<li><a title=\\"List of rivers of Iceland\\" href=\\"https://en.wikipedia.org/wiki/List_of_rivers_of_Iceland\\">List of rivers of Iceland</a></li>
<li><a class=\\"mw-redirect\\" title=\\"Volcanism in Iceland\\" href=\\"https://en.wikipedia.org/wiki/Volcanism_in_Iceland\\">Volcanism in Iceland</a></li>
<li><a class=\\"mw-redirect\\" title=\\"Waterfalls of Iceland\\" href=\\"https://en.wikipedia.org/wiki/Waterfalls_of_Iceland\\">Waterfalls of Iceland</a></li>
</ul>

<p>Source: <a href=\\"https://en.wikipedia.org/wiki/Gullfoss\\">Gullfoss</a>.<br>
Category: <a href=\\"https://demo.worona.org/wp-cat/travel/\\">Travel</a></p>
<blockquote class=\\"twitter-tweet\\"><p lang=\\"en\\" dir=\\"ltr\\">Exploring Iceland <a href=\\"https://t.co/G5c2iVAKHg\\">pic.twitter.com/G5c2iVAKHg</a></p>
<p>‚Äî Traveling Ideas (@Travel_Places_) <a href=\\"https://twitter.com/Travel_Places_/status/911998747198009344?ref_src=twsrc%5Etfw\\">September 24, 2017</a></p></blockquote>


<blockquote class=\\"instagram-media\\" style=\\"background:#FFF;border:0;margin:1px;max-width:658px;padding:0;width:99.375%;\\"><div style=\\"padding:8px;\\">
<div style=\\"background:#F8F8F8;line-height:0;margin-top:40px;padding:50% 0;text-align:center;width:100%;\\">

</div>
<p style=\\"margin:8px 0 0 0;padding:0 4px;\\"> <a href=\\"https://www.instagram.com/p/BdqJaLiAYfe/\\" style=\\"color:#000;font-family:Arial, sans-serif;font-size:14px;font-style:normal;font-weight:normal;line-height:17px;text-decoration:none;\\">Presents PHOTOGRAPHER | @sentinelphotography LOVES OF THE DAY January 7, 2018 LOCATION | #Gullfoss Waterfall SELECTED BY ADMIN | Linda ‚Äì @photobylindamoelgaard OUR TAG | #LOVES_UNITED_SCANDINAVIA VISIT PARTNERS COMMUNITY: * @loves_united_family * @loves_united_life * @loves_united_denmark * @loves_united_norway * @loves_united_finland * @loves_united_copenhagen * @traveling__night * @rsa_light_ * @igworld_global * @ig_eternity * OUR MAIL | loves.members@gmail.com * #lo_un_sc_sentinelphotography * #l_u_t_c_e_2018_january _________________________________________ #bestofscandinavia #igscandinavia #visitscandinavia #nordic2day #unlimitedscandinavia #scandinavia #ig_week_scandinavia #loves_united_life #loves_united_europe #exceptional_pictures #allbeauty_addiction #marvelouz_world #igworld_global #earth_shotz #ig_eternity #thehub_landscape #loves_united_landscape #loves_united_earth #tv_landscapes #bns_landscape #planet_earth_shots #zonephotographer #magic_marvels #earth_deluxe #landscape #landscapelover</a></p>
<p style=\\"color:#c9c8cd;font-family:Arial, sans-serif;font-size:14px;line-height:17px;margin-bottom:0;margin-top:8px;overflow:hidden;padding:8px 0 7px;text-align:center;white-space:nowrap;\\">A post shared by <a href=\\"https://www.instagram.com/loves_united_scandinavia/\\" style=\\"color:#c9c8cd;font-family:Arial, sans-serif;font-size:14px;font-style:normal;font-weight:normal;line-height:17px;\\"> LOVES ‚òÜ  AMAZING SCANDINAVIA</a> (@loves_united_scandinavia) on Jan 7, 2018 at 10:37am PST</p>
</div>
</blockquote>

<div id=\\"gallery-1\\" class=\\"gallery galleryid-60 gallery-columns-3 gallery-size-thumbnail\\"><figure class=\\"gallery-item\\"><div class=\\"gallery-icon landscape\\">
				<a href=\\"https://demo.worona.org/the-beauties-of-gullfoss/iceland-test/\\"><img data-attachment-id=\\"62\\" width=\\"150\\" height=\\"150\\" src=\\"https://demo.worona.org/wp-content/uploads/2016/11/Iceland-test-150x150.jpg\\" class=\\"attachment-thumbnail size-thumbnail\\" alt=\\"iceland\\"></a>
			</div></figure><figure class=\\"gallery-item\\"><div class=\\"gallery-icon landscape\\">
				<a href=\\"https://demo.worona.org/sunrise-at-phu-quoc-island/pexels-photo-225203/\\"><img data-attachment-id=\\"84\\" width=\\"150\\" height=\\"150\\" src=\\"https://demo.worona.org/wp-content/uploads/2016/11/pexels-photo-225203-150x150.jpeg\\" class=\\"attachment-thumbnail size-thumbnail\\" alt=\\"pexels-photo-225203\\"></a>
			</div></figure><figure class=\\"gallery-item\\"><div class=\\"gallery-icon landscape\\">
				<a href=\\"https://demo.worona.org/somewhere-in-romania/romania/\\"><img data-attachment-id=\\"13\\" width=\\"150\\" height=\\"150\\" src=\\"https://demo.worona.org/wp-content/uploads/2016/11/Romania-150x150.jpg\\" class=\\"attachment-thumbnail size-thumbnail\\" alt=\\"romania\\"></a>
			</div></figure><figure class=\\"gallery-item\\"><div class=\\"gallery-icon landscape\\">
				<a href=\\"https://demo.worona.org/banff-national-park/national-park/\\"><img data-attachment-id=\\"95\\" width=\\"150\\" height=\\"150\\" src=\\"https://demo.worona.org/wp-content/uploads/2016/11/national-park-150x150.jpg\\" class=\\"attachment-thumbnail size-thumbnail\\" alt=\\"national-park\\"></a>
			</div></figure><figure class=\\"gallery-item\\"><div class=\\"gallery-icon landscape\\">
				<a href=\\"https://demo.worona.org/switzerlands-mountains/lake/\\"><img data-attachment-id=\\"118\\" width=\\"150\\" height=\\"150\\" src=\\"https://demo.worona.org/wp-content/uploads/2016/11/lake-150x150.jpg\\" class=\\"attachment-thumbnail size-thumbnail\\" alt=\\"lake\\"></a>
			</div></figure></div>

",
  "creationDate": 1480098671000,
  "excerpt": "<p>Gullfoss is a waterfall located in the canyon of the Hv√≠t√° river in southwest Iceland.</p>
",
  "featured": 62,
  "headMeta": Object {
    "title": "The Beauties of Gullfoss - Demo Worona",
  },
  "id": 60,
  "link": "https://demo.worona.org/the-beauties-of-gullfoss/",
  "modificationDate": 1521738595000,
  "mst": "single",
  "parent": undefined,
  "slug": "the-beauties-of-gullfoss",
  "target": undefined,
  "taxonomies": Object {
    "category": Array [
      3,
      8,
    ],
    "latest": Array [
      "post",
    ],
    "tag": Array [
      10,
      9,
      13,
      11,
    ],
  },
  "title": "The Beauties of Gullfoss",
  "type": "post",
}
`;

exports[`Convert tag 1`] = `
Object {
  "headMeta": Object {
    "title": "Iceland Archives - Demo Worona",
  },
  "id": 9,
  "link": "https://demo.worona.org/tag/iceland/",
  "mst": "taxonomy",
  "name": "Iceland",
  "slug": "iceland",
  "target": undefined,
  "type": "tag",
}
`;
